src/js/filter.js

```diff
@@ -14,7 +14,8 @@ class Filter {
     this.$controls = document.getElementById('filter-controls')
     this.renderControls(this.$controls)
     this.list = new List(this.$el.id, {
-      valueNames: this.sortOptions.map(o => o.name)
+      valueNames: [...this.sortOptions.map(o => o.name), ...this.searchOptions.searchOn],
+      fuzzySearch: this.searchOptions.fuzzySearch
```

Comment:
```suggestion
      search: this.searchOptions.search
```

See previous comment about replacing fuzzy search with the default search for list.js